harness.sh

```diff
@@ -485,16 +485,25 @@ function typeset_and_compare {
         else
             if $verify "$texfile"
             then
-                if $skip_cache || [[ "$pdffile" -nt "$IMAGE_CACHE/$indir/$outdir" ]]
+                directory="$IMAGE_CACHE/$indir/$outdir"
+                not_nice=false
+                if $skip_cache || [[ "$pdffile" -nt "$directory" ]]
                 then
-                    rm -fr "$IMAGE_CACHE/$indir/$outdir" && \
-                    mkdir -p "$IMAGE_CACHE/$indir/$outdir" && \
+                    rm -fr "$directory" && \
+                    mkdir -p "$directory" && \
                         convert -background white -alpha remove \
                             -colorspace Gray -channel R -separate \
                             -density $PDF_DENSITY "$pdffile" \
-                            "$IMAGE_CACHE/$indir/$outdir/page-%d.png" || \
+                            "$directory/page-%d.png" || \
+                        not_nice=true
```

Comment:
The indentation of this line is throwing me off; it might just be the way github is showing it, but I would think this should align with the line above it since this is the next argument of the `||` operator, just like the previous line is.

Reply:
Looking at it now, I think the problem is actually several lines above in line 493.  Line 492 starts a long line continuation with several boolean operators: `rm... && mkdir... && convert... || not_nice...`.  Thus 493 (`mkdir...`), 494 (`convert...`) and 498  (`not_nice...`) should all be at the same indentation level (493 currently isn't).  Lines 496 and 497 are part of the `convert...` and thus have additional indentation to indicate that we haven't yet finished with that particular part of the boolean expression.

Does that make sense?